Sources/eppo/UniversalFlagConfig.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+public struct UniversalFlagConfig : Decodable {
+    let createdAt: Date?;
+    let flags: [String: UFC_Flag];
+    // todo: add bandits
+    
+    static func decodeFromJSON(from jsonString: String) throws -> UniversalFlagConfig {
+        let decoder = JSONDecoder()
+
+        // Set up the date formatter
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"  // Adjusted to include milliseconds
```

Comment:
the UFC API returns dates with milliseconds:

```
new Date().toISOString()
```

<img width="200" alt="Screenshot 2024-05-31 at 11 48 07 PM" src="https://github.com/Eppo-exp/eppo-ios-sdk/assets/57361/3733239d-ba8e-484e-be08-c073cbb35c8e">